packages/scid/scid.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Vincent Bernardoff <vb@luminar.eu.org>"
+authors: "Vincent Bernardoff <vb@luminar.eu.org>"
+homepage: "https://github.com/vbmithr/scid"
+bug-reports: "https://github.com/vbmithr/scid/issues"
+license: "ISC"
+doc: "https://vbmithr.github.io/scid/doc"
+dev-repo: "https://github.com/vbmithr/scid.git"
+build: ["ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%"]
+build-test: [
+  ["ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "true"]
+  ["ocaml" "pkg/pkg.ml" "test"]
+]
+depends: [
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build}
+  "ounit" {test}
+  "result"
+  "ocplib-endian"
+]
+available: [ocaml-version >= "4.01.0"]
```

Comment:
The build's [failing](https://travis-ci.org/ocaml/opam-repository/jobs/203873044) because the code uses `String.init`, which was introduced in OCaml 4.02:

```
# File "src/scid.ml", line 24, characters 14-25:
# Error: Unbound value String.init
# Command exited with code 2.
```

Could you please tighten the `ocaml-version` constraint?